### ISSUE TITLE
got the various files when executre vidclip 

### ISSUE THREAD

**Issue by sugizo** [zackees/zcmds#11]
**execute**
`!vidclip --start_timestamp 0 --end_timestamp 3 --outname 'vidclip.mp4' --crf 23 'ForBiggerBlazes.mp4'`

**result** 
got the various files
**e.g.**
```
rw-r--r-- 1 root root  320950 Apr 12 01:19 vidclip_alt.mp4
-rw-r--r-- 1 root root  320950 Apr 12 01:18 vidclip.mp4
```

**expected result**
just the mention file when execute the command 'vidclip.mp4'

**Comment by zackees** [comment]
Could not reproduce. Please provide a repro case.

![image](https://github.com/zackees/zcmds/assets/6856673/c257abc1-e9d2-4259-b81d-ffc243b8e4df)

**Comment by zackees** [comment]
This seems to be working as intended.

**Comment by sugizo** [comment]
no, got this after upgrade it
you need to run it twice, 
solution 
perhaps use -y argument to static_ffmpeg 
e.g.
`static_ffmpeg -y`

![image](https://github.com/zackees/zcmds/assets/2418295/1ea1f13e-bd85-4e09-9d6d-d17d3294a71b)

**Comment by zackees** [comment]
Fixed in version `1.4.100`
